app/models/sac_cas/event/participation.rb

```diff
@@ -12,6 +12,11 @@ module SacCas::Event::Participation
     before_save :update_previous_state, if: :state_changed?
   end
 
+  def particpant_cancelable?
+    event.applications_cancelable? && event.state != 'annulled' &&
+      event.dates.map(&:start_at).min.future?
```

Comment:
Die Methode könnte auch direkt auf den Kurs, braucht ja keine Participant Infos.

Reply:
Ich lass sie auf der Participation, dann funktionierts auch falls wir da mal eine `Event` und keine `Event::Course` in der Hand haben, bei dem STI zeugs bin ich mir nie ganz sicher :confused:

Reply:
`Event` haben aber eigentlich keinen `state`. Aber es könnte natürlich auch auf den `Event`